cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AuthorizationHandler.java

```diff
@@ -123,16 +129,15 @@ public void revoke(HttpRequest httpRequest, HttpResponder httpResponder) throws
       throw new BadRequestException("Missing request body");
     }
 
-    if (request.getUser() == null && request.getActions() == null) {
-      auth.revoke(request.getEntity());
```

Comment:
```
    if (!enabled) {
      httpResponder.sendStatus(HttpResponseStatus.NOT_FOUND);
      return;
    }

    CheckAuthorizedRequest request = parseBody(httpRequest, CheckAuthorizedRequest.class);
    if (request == null) {
      throw new BadRequestException("Missing request body");
    }
```
This check part is repeated in all handler. It will be nice to extract it out in a validation/check method.